src/GraphQL.Tests/Bugs/Bug1772.cs

```diff
@@ -0,0 +1,75 @@
+using GraphQL.Execution;
+using GraphQL.SystemTextJson;
+using GraphQL.Types;
+using GraphQL.Validation;
+using GraphQL.Validation.Complexity;
+using Shouldly;
+using System;
+using System.Linq;
+using System.Numerics;
+using System.Threading.Tasks;
+using Xunit;
```

Comment:
It seems to be unused usings.

Reply:
Wasn't sure if you cared since it was in a test.  No problem, I will keep them clean in tests from now on.

Reply:
I care about it.